src/vitabel/timeseries.py

```diff
@@ -65,10 +65,22 @@ def _timeseries_list_info(series_list: list[TimeSeriesBase]) -> pd.DataFrame:
                 "Offset": series.offset,
             }
         )
+
+        if hasattr(series, "anchored_channel"):
+            if series.anchored_channel is not None:
+                info_dict[idx]["Attached Channel"] = series.anchored_channel.name #name of the channel
+            else:
+                info_dict[idx]["Attached Channel"] = ""
+        if hasattr(series, "labels"):
+            info_dict[idx]["Attached Labels"] =  ", ".join(
+                [label.name for label in series.labels]
+            )
+
     df = pd.DataFrame(info_dict).transpose()
     df_columns = ['Name', 'Length', 'First Entry', 'Last Entry', 'Offset']
-    metadata_columns = [col for col in df.columns.values if col not in df_columns]
-    return df.reindex(columns=df_columns + metadata_columns)
+    attach_columns = ["Attached Channel", "Attached Labels"]
+    metadata_columns = [col for col in df.columns.values if col not in df_columns+ attach_columns]
```

Comment:
why would the metadata contain `"Attached Channel"` or `"Attached Labels"` fields? things we already keep track of should not be duplicated in the metadata.

```suggestion
    metadata_columns = [col for col in df.columns.values if col not in df_columns]
```

Reply:
actually also done in [b4929b8](https://github.com/UniGrazMath/vitabel/pull/214/commits/b4929b8211a093a506be6b04d3171306d5d72c15)